src/p11_child/p11_child_common.c

```diff
@@ -204,6 +205,8 @@ int main(int argc, const char *argv[])
          _("PKCS#11 URI to restrict selection"), NULL},
         {"chain-id", 0, POPT_ARG_LONG, &chain_id,
          0, _("Tevent chain ID used for logging purposes"), NULL},
+        {"timeout", 0, POPT_ARG_LONG, &timeout,
+         0, _("Network timeout"), NULL},
```

Comment:
I would be more specific and would mention OSCP.
Maybe something like "OCSP server communication timeout".

Reply:
done